tech/database/mariadb/about.md

```diff
@@ -218,3 +218,19 @@ $ dnf install mariadb-oqgraph-engine
 In Fedora, MariaDB server is available also as a dynamic library, that can be handy in some applications. This library (`libmysqld.so`) is available in the package `mariadb-embedded` and header files for building an application against this library are available in the package `mariadb-embedded-devel`.
 
 However the use of the embedded library is discouraged. MySQL 8 dropped the embedded library and I expect MariaDB to go in the same direction.
+
+# Troubleshooting:
+* Fix Access denied for user 'root'@'localhost' for phpMyAdmin
```

Comment:
This would go to a different section, though. Probably to `Web Apps` or to `PHP`-related stuff (did not look up that now, please give it a look yourself).

Reply:
Then please create a stand-alone page in MariaDB subsection. We try to make it as simple as possible and not everyone will use phpMyAdmin.

Reply:
okay.

Reply:
Let's remove this then.